File: TP3-CPP/Kode Program/JadwalTrip.cpp
#pragma once
#include <string>
#include <iostream>
using namespace std;

// Kelas JadwalTrip menyimpan informasi tentang jadwal keberangkatan dan kepulangan trip
class JadwalTrip {
public:
    string tgl_mulai;
    string tgl_selesai;
    string jam_berangkat;

    // Konstruktor untuk menginisialisasi semua atribut
    JadwalTrip(string tgl_mulai, string tgl_selesai, string jam_berangkat)
        : tgl_mulai(tgl_mulai), tgl_selesai(tgl_selesai), jam_berangkat(jam_berangkat) {}

    // Method untuk menampilkan detail jadwal trip ke layar
    void tampilkan() const {
        cout << "Tanggal Mulai: " << tgl_mulai << ", Tanggal Selesai: " << tgl_selesai
             << ", Jam Berangkat: " << jam_berangkat << endl;
    }
};

File: TP3-Java/Kode Program/JadwalTrip.java
public class JadwalTrip {
    private String tglMulai;
    private String tglSelesai;
    private String jamBerangkat;

    /*Konstruktor untuk menginisialisasi objek JadwalTrip  */
    public JadwalTrip(String tglMulai, String tglSelesai, String jamBerangkat) {
        this.tglMulai = tglMulai;
        this.tglSelesai = tglSelesai;
        this.jamBerangkat = jamBerangkat;
    }

    /*menampilkan informasi jadwal trip */
    public void tampilkan() {
        System.out.println("Tanggal Mulai: " + tglMulai +
                           ", Tanggal Selesai: " + tglSelesai +
                           ", Jam Berangkat: " + jamBerangkat);
    }
}

